style/icons.css

```diff
@@ -86,3 +86,46 @@
 [data-jp-theme-light='false'] .jp-ViewBreakpointIcon {
   background-image: url('icons/md-dark/view-breakpoint.svg');
 }
+
+.jp-debugger-switch-button::before {
+  content: '';
+  width: 14px;
+  height: 14px;
+  background-position: left center;
+  background-repeat: no-repeat;
+  background-size: 14px;
+  margin-right: 4px;
+  background-image: var(--jp-icon-bug);
+}
+
+.jp-toggle-switch {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  background-color: var(--jp-border-color1);
+  -webkit-transition: 0.4s;
+  transition: 0.4s;
+  border-radius: 34px;
+  width: 35px;
+}
+
+.jp-toggle-switch::before {
+  content: '';
+  height: 10px;
+  width: 10px;
+  margin-left: 5px;
+  margin-right: 50%;
+  background-color: white;
+  -webkit-transition: 0.4s;
+  transition: 0.4s;
+  border-radius: 50%;
+}
+
+[data-jp-debugger='true'] .jp-toggle-switch {
+  background-color: #f17c00;
```

Comment:
Maybe we could use a lab variable here? Either the `--jp-warn-color*` or `--md-orange-***`